lib/theme_check/theme.rb

```diff
@@ -4,19 +4,33 @@
 module ThemeCheck
   class Theme
     DEFAULT_LOCALE_REGEXP = %r{^locales/(.*)\.default$}
-    attr_reader :root
+    LIQUID_REGEX = /\.liquid$/i
+    JSON_REGEX = /\.json$/i
 
-    def initialize(root, ignored_patterns: [])
-      @root = Pathname.new(root)
-      @ignored_patterns = ignored_patterns
+    # temp
```

Comment:
Why temp?

Reply:
If I recall correctly, I only wanted to expose this while unit testing and shouldn't be part of the public API. I left the note there to remind me to remove it :p Clearly I didn't remember :)